### PR TITLE
feat: link project page admin to profile

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: regen-network/regen-registry#1648

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. For a profile that does have info: https://deploy-preview-1886--regen-registry.netlify.app/project/C45-010

Screen Shot 2023-04-27 at 10 59 32 AM

2. For a profile that has not filled anything out yet: https://deploy-preview-1886--regen-registry.netlify.app/project/C45-011

Screen Shot 2023-04-27 at 11 01 28 AM

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.utils.ts [hunk 1]
```diff
@@ -14,21 +18,27 @@ import { getAreaUnit, qudtUnit } from 'lib/rdf';
 // TODO
 // This is a temporary hack to show Regen as a Project Admin when applicable
 
-const addressesMap = [
-  'regen123a7e9gvgm53zvswc6daq7c85xtzt8263lgasm', // Mainnet - Credit classes
-  'regen1v2ncquer9r2ytlkxh2djmmsq3e8we6rjc9snfn', // Mainnet - Projects
-  'regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46', // Redwood - Shared dev account
-];
+// TODO: delete this after the profile info is updated where needed..
```
Comment (wgwz) [inline web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.utils.ts:21]: TODO

**Comment by wgwz** [comment]
hey @erikalogie i just wanted to ask you for some testing on this, i put some instructions in the description of the PR under "how to test". LMK what you think, and if you have any q's

**Comment by erikalogie** [comment]
@wgwz somehow I missed this notification, I'll test now

**Comment by erikalogie** [comment]
@wgwz this looks like it is working well.

A few points of feedback: I think that the link should open in the same window/tab, not a new one.

It is a bit weird that the preview image on the project page in the case that there is no profile look like this: https://monosnap.com/file/HYyaVUtYI3irjFDKP9sMKWMDlSGoP9 but then on the profile it looks like this: https://monosnap.com/file/VD5I8JkETFCb287PVQVzIJftvycsjG. This would be solved by implementing this task: https://app.zenhub.com/workspaces/regen-registry-5f8998bec8958d000f4609e2/issues/gh/regen-network/regen-registry/1280 but I guess we won't prioritize this until later.

I think that we should we just use the default avatar for the individual or org here if they have not edited their profile avatar. I guess we'd have to make an exception for cases like this one: https://app.regen.network/project/C02-002 where we seemingly have hard-coded a link to the project developer's website. In that case it should show the org avatar.

Both of these default images should look like this:
Avatar default company
Avatar default

**Comment by wgwz** [comment]
@erikalogie thanks for the feedback!
i made some changes and as i was testing locally i thought it'd be helpful to write out these scenarios.

do these all look right to you?

**Case 1**: an user-type admin, that has not set up profile

Screen Shot 2023-05-03 at 6 28 02 PM

**Case 2**: an org-type admin, that has not set up profile

Screen Shot 2023-05-03 at 6 30 20 PM

**Case 3**: an admin that has no profile whatsoever (we make it look like a "user")

Screen Shot 2023-05-03 at 6 32 05 PM

**Case 4**: and of course, an admin that does have some profile info

Screen Shot 2023-05-03 at 6 39 04 PM

**Comment by wgwz** [comment]
> Case 3: an admin that has no profile whatsoever (we make it look like a "user")

FYI - my screenshot for this case shows a caching issue, just ignore this one for now. i’ll update it soon.

**Comment by erikalogie** [comment]
@wgwz this looks good except that I believe case 3 should look like case 2.

**Comment by wgwz** [comment]
> Looks good! Let's not forget about the TODO in `getDisplayAdmin` before merging.

@flagrede could you give me some pointers for how to type that `party`? i was struggling to figure out how to type it without having some ridiculously huge type based on what my editor was showing me about the party object.

**Comment by wgwz** [comment]
> > Case 3: an admin that has no profile whatsoever (we make it look like a "user")
> 
> FYI - my screenshot for this case shows a caching issue, just ignore this one for now. i’ll update it soon.

nevermind, the screenshot for case 3 is fine. this scenario is where i'm signed in to an org account, viewing a project whose admin does not have a profile.

**Comment by wgwz** [comment]
> @wgwz this looks good except that I believe case 3 should look like case 2.

just noting that i spoke with @erikalogie about this and case 3 looks as it should. there was some confusion between what is shown here, vs. what we want to show for "project developer" rather than "project admin". more to come on that.

**Comment by wgwz** [comment]
> > Looks good! Let's not forget about the TODO in `getDisplayAdmin` before merging.
> 
> @flagrede could you give me some pointers for how to type that `party`? i was struggling to figure out how to type it without having some ridiculously huge type based on what my editor was showing me about the party object.

ah, ok. so i'm pretty sure i've got it now, just by luck. `Maybe<PartyFieldsFragment>`.

**Comment by wgwz** [comment]
Regarding displaying information for project developers.

**Case 1**: if the project developer has a profile and they have uploaded some information there.
https://user-images.githubusercontent.com/10120306/236313855-a1be2f00-71c6-4b03-a176-60b09b189dd3.jpg

**Case 2**: the project developer has an empty, user-type profile.
https://user-images.githubusercontent.com/10120306/236314404-5b5eee29-07b6-4569-8348-92adb0ae5d7a.jpg

**Case 3**: the project developer has an empty, org-type profile.
https://user-images.githubusercontent.com/10120306/236314640-6f2571cb-e3aa-4696-a0f0-04019cb25063.jpg

**Case 4**: the project developer does not have a profile (nothing shows up, expected)
https://user-images.githubusercontent.com/10120306/236315101-5ac2592c-42d4-43d7-95f2-df0af8794aa7.jpg

**Case 5**: the project developer does not have a profile, but the anchored metadata has a project developer and some basic info (note: we're using the correct org icon here now)
https://user-images.githubusercontent.com/10120306/236322214-16dad76e-03a3-4cd4-a74f-244137f07db8.jpg

/cc @erikalogie testing is complicated for this one, LMK what you think. i believe i covered everything you asked for. Case 5 shows the fix for the org icon issue you pointed out here: https://app.regen.network/project/C02-002

**Comment by erikalogie** [comment]
> Regarding displaying information for project developers.
> 
> **Case 1**: if the project developer has a profile and they have uploaded some information there. https://user-images.githubusercontent.com/10120306/236313855-a1be2f00-71c6-4b03-a176-60b09b189dd3.jpg
> 
> **Case 2**: the project developer has an empty, user-type profile. https://user-images.githubusercontent.com/10120306/236314404-5b5eee29-07b6-4569-8348-92adb0ae5d7a.jpg
> 
> **Case 3**: the project developer has an empty, org-type profile. https://user-images.githubusercontent.com/10120306/236314640-6f2571cb-e3aa-4696-a0f0-04019cb25063.jpg
> 
> **Case 4**: the project developer does not have a profile (nothing shows up, expected) https://user-images.githubusercontent.com/10120306/236315101-5ac2592c-42d4-43d7-95f2-df0af8794aa7.jpg
> 
> **Case 5**: the project developer does not have a profile, but the anchored metadata has a project developer and some basic info (note: we're using the correct org icon here now) https://user-images.githubusercontent.com/10120306/236322214-16dad76e-03a3-4cd4-a74f-244137f07db8.jpg
> 
> /cc @erikalogie testing is complicated for this one, LMK what you think. i believe i covered everything you asked for. Case 5 shows the fix for the org icon issue you pointed out here: https://app.regen.network/project/C02-002

This looks good to me!

**Pull request merged**